docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
+cat <<EOF | kubectl create -f -
 apiVersion: sources.knative.dev/v1beta2
 kind: PingSource
 metadata:
   name: test-ping-source-binary
 spec:
-  schedule: "*/1 * * * *"
+  schedule: "*/2 * * * *"
   contentType: "text/plain"
   dataBase64: "ZGF0YQ=="
   sink:
     ref:
-      apiVersion: v1
+      apiVersion: serving.knative.dev/v1
       kind: Service
       name: event-display
 EOF
 ```
 
-### Verify
+## Verify the message was sent
+
+Verify that the message was sent to the Knative Eventing system by looking at message dumper logs.
+
+{{< tabs name="view-event" default="kubectl" >}}
+{{% tab name="kubectl" %}}
+
+View the logs of the event-display service by running:
+
+```shell
+kubectl logs -l serving.knative.dev/service=event-display -c user-container --since=10m
+```
+
+{{< /tab >}}
+{{% tab name="kail" %}}
 
-View the logs for the `event-display` event consumer by
-entering the following command:
+Use `kail` to tail the logs of the subscriber by running:
 
 ```shell
-kubectl -n pingsource-example logs -l app=event-display --tail=100
+kail -l serving.knative.dev/service=event-display -c user-container --since=10m
 ```
 
-This returns the `Attributes` and `Data` of the events that the PingSource sent to the `event-display` Service:
+For more information, see the [`kail`](https://github.com/boz/kail) repository in GitHub.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+
+Log entries appear showing the request headers and body from the source:
 
 ```shell
 ☁️  cloudevents.Event
 Validation: valid
 Context Attributes,
   specversion: 1.0
   type: dev.knative.sources.ping
-  source: /apis/v1/namespaces/pingsource-example/pingsources/test-ping-source
-  id: 49f04fe2-7708-453d-ae0a-5fbaca9586a8
-  time: 2021-03-25T19:41:00.444508332Z
+  source: /apis/v1/namespaces/default/pingsources/test-ping-source
+  id: d8e761eb-30c7-49a3-a421-cd5895239f2d
+  time: 2019-12-04T14:24:00.000702251Z
   datacontenttype: application/json
 Data,
   {
     "message": "Hello world!"
   }
 ```
 
-If you created a PingSource with binary data, you should also see the following:
+If you created a PingSource with binary data, you also see the following:
 
 ```shell
 ☁️  cloudevents.Event
 Validation: valid
 Context Attributes,
   specversion: 1.0
   type: dev.knative.sources.ping
-  source: /apis/v1/namespaces/pingsource-example/pingsources/test-ping-source-binary
-  id: ddd7bad2-9b6a-42a7-8f9b-b64494a6ce43
-  time: 2021-03-25T19:38:00.455013472Z
+  source: /apis/v1/namespaces/default/pingsources/test-ping-source-binary
+  id: a195be33-ff65-49af-9045-0e0711d05e94
+  time: 2020-11-17T19:48:00.48334181Z
   datacontenttype: text/plain
 Data,
-  data
+  ZGF0YQ==
 ```
 
-### Cleanup
+## Clean up
+
+Delete the PingSource instance:
 
-Delete the `pingsource-example` namespace and all of its resources from your
-cluster by entering the following command:
+{{< tabs name="delete-source" default="kubectl" >}}
+{{% tab name="kubectl" %}}
+
+Delete the PingSource instance by running:
```

Comment:
```suggestion
Delete the PingSource instance:
```

Reply:
I think "by running" matters because it better orients readers, especially blind ones. I've written "Documenting Commands" rules in the Style Guide given Google's Style Guide is a little light on them.